core/indexer/elasticsearch_test.go

```diff
@@ -53,15 +56,6 @@ func TestElasticIndexer_NewIndexerWithNilUrlShouldError(t *testing.T) {
 	require.Equal(t, core.ErrNilUrl, err)
 }
 
-func TestElasticIndexer_NewIndexerWithNilShardCoordinatorShouldError(t *testing.T) {
```

Comment:
there are no tests for the newly added NodesCoordinator and EpochStartNotifier.

Reply:
done.

Reply:
ok